src/Elastic.Apm/Api/ISpan.cs

```diff
@@ -46,6 +46,11 @@ public interface ISpan
 		/// </summary>
 		Dictionary<string, string> Tags { get; }
 
+		/// <summary>
+		/// A list of user defined custom fields.
+		/// </summary>
+		Dictionary<string, object> Custom { get; }
```

Comment:
There is no `custom` on span. We only have this on the transaction's context. See [here](https://github.com/elastic/apm-server/blob/master/docs/spec/spans/span.json).

Reply:
Reverted changes with custom